core/lib/mno_enterprise/concerns/controllers/auth/registrations_controller.rb

```diff
@@ -46,10 +46,13 @@ module ClassMethods
   def create
     build_resource(sign_up_params)
     resource.password ||= Devise.friendly_token
-    
+
     resource_saved = resource.save
-    
+
     if resource_saved
+
+      MnoEnterprise::EventLogger.info('user_add', resource_saved.id, 'User Signup', OpenStruct.new(id: 0 ), {email: resource_saved.email})
```

Comment:
Wouldn't `resource` be an instance of `MnoEnterprise::User`?
If so you could use:
```ruby
MnoEnterprise::EventLogger.info('user_add', resource_saved.id, 'User Signup', resource_saved)
```